cgk_calcium_tools/filehandler.py

```diff
@@ -609,33 +565,37 @@ def run_step(
         assert len(operation) == 1, f"Step {op} not starts with {steps_list}."
         operation = operation[0]
 
-        parameters = self.default_parameters[operation].copy()
-        for key, value in kws.items():
-            assert key in parameters, f"The parameter: {key} does not exist"
-            parameters[key] = value
-
+        if not op.startswith('DI'):
+            parameters = self.default_parameters[operation].copy()
+            for key, value in kws.items():
+                assert key in parameters, f"The parameter: {key} does not exist"
+                parameters[key] = value
+        
+        #TASK: No need for this step at this point - Can you confirm Fernando?
+        '''
         if self.processing_steps.index(op) == 0:  # if first step
             # check if all exist
             for finput, _ in self.get_pair_filenames(op):
                 assert os.path.exists(
                     finput
                 ), f"""File {finput} not exist:
                     Run .de_interleave() to De-interleave multiplane movies"""
+        '''
 
```

Comment:
Is needed if the user doesn't include the DI step and tried to run the pipeline having mutiplane videos. It should be a bit different now, because the first step is DI.

A patch could be just run that code if the user doesn't have DI as the first step